src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -134,15 +139,16 @@ trait CromwellApiService extends HttpService with PerRequestCreator  {
       }
     }
 
+  @Path("/{version}/{id}/outputs")
   @ApiOperation(
-    value = "Query for workflow outputs based on workflow id.",
+    value = "Query for workflow outputs based on workflow id",
     nickname = "outputs",
     httpMethod = "GET",
     produces = "application/json"
-    //,response = classOf[WorkflowOutputs]
   )
   @ApiImplicitParams(Array(
-    new ApiImplicitParam(name = "id", required = true, dataType = "string", paramType = "path", value = "workflow")
+    new ApiImplicitParam(name = "version", required = true, dataType = "string", paramType = "path", value = "API Version", allowableValues = CromwellApiService.VersionAllowableValues),
+    new ApiImplicitParam(name = "id", required = true, dataType = "string", paramType = "path", value = "Workflow Identifier")
   ))
   @ApiResponses(Array(
     new ApiResponse(code = 200, message = "Successful Request"),
```

Comment:
Can the `response` attribute that was removed from the `@ApiOperation` be relocated to the appropriate `@ApiResponse` here as it was in other APIs?  If I recall Kristian was having compilation problems around this annotation.  :frowning: